### DISCUSSION TITLE
net module not found when importing

### DISCUSSION THREAD

**Discussion by awidearray** [rainbow-me/rainbowkit#1300] [Q&A] [answered]
./node_modules/ws/lib/sender.js:5:0
Module not found: Can't resolve 'net'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./node_modules/ws/index.js
./node_modules/@walletconnect/legacy-client/dist/esm/socket.js
./node_modules/@walletconnect/legacy-client/dist/esm/core.js
./node_modules/@walletconnect/legacy-client/dist/esm/index.js
./node_modules/@walletconnect/legacy-provider/dist/esm/signer.js
./node_modules/@walletconnect/legacy-provider/dist/esm/index.js
./node_modules/@wagmi/connectors/dist/walletConnectLegacy.js
./node_modules/@wagmi/core/dist/connectors/walletConnectLegacy.js
./node_modules/wagmi/dist/connectors/walletConnectLegacy.js
./node_modules/@rainbow-me/rainbowkit/dist/index.js
./src/providers/walletprovider.tsx
./src/pages/_app.tsx

i think im close but need a little halp

**Comment by DanielSinclair** [comment, marked as answer]
Changes in WalletConnect require that `net` is polyfilled. Reference our docs [here](https://www.rainbowkit.com/docs/installation#react-nextjs-and-webpack) and Next.js example [here](https://github.com/rainbow-me/rainbowkit/tree/main/examples/with-next)
